web_bench/README.md

```diff
@@ -0,0 +1,14 @@
+This is a benchmark generating a proof for withdraw relation with WebAssembly threads.
+
+Install dependencies:
+`yarn install`
+
+Build webpage:
+`yarn build`
```

Comment:
it's sad that every time it rebuilds the whole rust source from scratch :|

Reply:
Agreed :/